src/markdown/markdown-body.scss

```diff
@@ -10,8 +10,6 @@
   line-height: $body-line-height;
   word-wrap: break-word;
 
-  @import "../base/kbd.scss"; // adds support for keyboard shortcuts
```

Comment:
Is importing a file like that not possible in dart-sass? If so, we might need to just duplicate those styles?

Reply:
<kbd>Hello inspect the styles on me</kbd>

yes, I meant to say. dart-sass didn't like this. but checking the `.markdown-body kbd` and the normal `kbd` that's included in base both apply in dotcom. So I'm unsure if this more specific import is needed?

If it is, we can try putting `.markdown-body kbd, kbd {}` in the base/kbd.scss file.